core/cas-server-core-util/src/main/java/org/apereo/cas/util/gen/Base64RandomStringGenerator.java

```diff
@@ -0,0 +1,33 @@
+package org.apereo.cas.util.gen;
+
+import org.apereo.cas.util.EncodingUtils;
+
+/**
+ * URL safe base64 encoding implementation of the RandomStringGenerator that allows you to define the
```

Comment:
Turn this into a @link javadoc tag please. Lest the file gets moved/renamed, we should be able to catch it.